Kinvey/Kinvey/RealmCache.swift

```diff
@@ -449,6 +449,14 @@ internal class RealmCache<T: Persistable>: Cache<T> where T: NSObject {
         }
     }
     
+    override func clear() {
```

Comment:
@heyzooi similar to my previous comment, there should be an optional query parameter here.